examples/manifests/kustomization.yaml

```diff
@@ -16,7 +16,10 @@ replicas:
 images:
 - name: cloud-event-proxy
   newName: quay.io/aneeshkp/cloud-event-proxy
```

Comment:
I think we can soon change this to the quay.io/openshift/cloud-event-proxy

Reply:
so no change for now?

Reply:
Nishanth has to fix that ,once he fix that we will do a PR to change all images to point to openshift quay

Reply:
but consumer (example) will still stay under private repo